urdf/CMakeLists.txt

```diff
@@ -118,6 +120,7 @@ ament_export_libraries(${PROJECT_NAME})
 ament_export_targets(${PROJECT_NAME})
 
 ament_export_dependencies(pluginlib)
+ament_export_dependencies(rcutils)
```

Comment:
We don't need to export a dependency on `pluginlib` or `rcutils` here, as they are both PRIVATE dependencies.

```suggestion
```

Reply:
https://github.com/ros2/urdf/pull/37/commits/e9419ebdf3a3fe600e587a0e437f66b56756ee74